03_backend_developement/01_HTTP/05_centralized_webservice/.test_utils/readcode.js

```diff
@@ -0,0 +1,26 @@
+/*
+  This file is for teachers to help with the tests.
+ */
+
+const fs = require("fs");
+
+module.exports = function(path) {
+  return new Promise(function(resolve, reject) {
+    fs.readFile(path, "utf8", function(err, text) {
+      if (err) {
+        reject(err);
+        return;
+      }
+      const studentCode = text
+        .replace(/\/\*([^]*?)\*\//gm, "") // multiline comments
+        .replace(/\/\/[^]*?\n/g, "") // single line comments
+        .trim();
+
+      if (studentCode.length) {
+        resolve(studentCode);
+      } else {
+        reject("File does not contain any code")
+      }
+    });
+  });
+};
```

Comment:
Did you change something from the original file?
All files that uses this code are currenlty looking for the same path (random example: `02_programming_fundamentals/03_algorithms_day_1/05_play_with_for_loop_bonus/__tests__/06_sea.test.js`) So I don't see reason to duplicate it if nothing's changed.

But rereading the code, do you still need this function? As you require the student's file and run their function directly, it should not be needed anymore.